azurelinuxagent/common/event.py

```diff
@@ -581,24 +580,23 @@ def _trim_extension_event_parameters(event):
         event.parameters = trimmed_params
 
     @staticmethod
-    def report_dropped_events_error(count, errors, op, max_errors_to_report): # pylint: disable=C0103
+    def report_dropped_events_error(count, errors, operation_name, max_errors_to_report):
         err_msg_format = "DroppedEventsCount: {0}\nReasons (first {1} errors): {2}"
         if count > 0:
-            add_event(op=op,
+            add_event(op=operation_name,
                       message=err_msg_format.format(count, max_errors_to_report, ', '.join(errors)),
                       is_success=False)
 
-    def collect_events(self): # pylint: disable=R0914
+    def collect_events(self, enqueue_event_func): # pylint: disable=R0914
```

Comment:
Not sure if you looked at this linter warning. If you did, can you add a comment explaining why it is disabled? If you do this, could you also add a space before the #pylint comment (inline comments are supposed to be preceded by 2 spaces and pycharm is very noisy about this)

Reply:
Sorry missed this one. Let me look it up

Reply:
Sorry missed this one. Added a comment!